Imabulary/Imabulary
# [Bugfix] [FE] Bug - user gets 401 Unauthorized error first time they enter the app (IMB-178)

## Overview
This PR will fix `401 Unauthorized` on initial login

## Test cases

1. Sign in with Google and use an email that doesn't exist in our internal DB
2. Verify that `401 Unauthorized` error doesn't appear when you log in

## Issue

closes [IMB-178](https://www.notion.so/FE-Bug-user-gets-401-Unauthorized-error-first-time-they-enter-the-app-699cdf1a1fa744e3b332b48c415b8f22?pvs=4)
